doc/part-1-github-ui/github-contributions.md

```diff
@@ -8,6 +8,27 @@ description: 'PART 1: GITHUB UI'
 
 At this point, the focus is not about Git and Github best practices. The learning videos here will cover the basic things needed to get started with contributing to any project using the Github UI. What you will learn will also help you to tackle the tasks that have been provided for you to do after watching the learning videos.
 
+You will learn the following in this section:
+
+1. Forking a repository and how to know which branch you are in. 
+   
+2. Contributions (overview, issues, commits) and branching. 
+   
+3. How to make pull request, and link an issue to pull request. 
+   
+4. Reviews ( branch, pull requests, linking issues continued).
+   
+5. Managing files or folders (create, edit and delete). 
+    
+6. Branches (overview, deleting branches connected to pull request or fork).
+ 
+7. When to delete a branch connected to pull request or a fork. 
+  
+8. Syncing your fork.
+9. General information about the training tasks.
+
+{% embed url="https://youtu.be/4Z0Z1Z0Z1Z0" %}
```

Comment:
@aaryansr26 You added this particular video embed by mistake. Please remove it.

Reply:
Done! @Ifycode